pom.xml

```diff
@@ -127,20 +127,6 @@
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>com.jayway.restassured</groupId>
```

Comment:
pom.xml had multiple dependencies to restassured for some reason, I remove the older version so that it doesn't conflict with newer one.

Reply:
👍